### ISSUE TITLE
Cannot interact with NewPipe in Split Screen Mode

### ISSUE THREAD

**Issue by Dakkaron** [TeamNewPipe/NewPipe#7958]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.22.1

### Steps to reproduce the bug

On Android 11
1. Open a video
2. Open split screen mode
3. Touch the part of the screen where Newpipe is running or press back

### Expected behavior

Video is running, Player should react to interactions

### Actual behavior

Video is running, but the app doesn't react to any interactions at all until the split screen is closed and only newpipe is open.

### Affected Android/Custom ROM version

Android 11

### Affected device model

Fairphone 4

### Additional information

Seems to be a regression. Worked in 0.21.X

**Comment by litetex** [comment]
Duplicate of #7711

**Comment by Dakkaron** [comment]
No, thats two distinct problems. In #7711 the video is black but the player is interactable. That occurred in 0.21.X on Android 11 after resizing the split screen mode. That issue is still around in 0.22.1.


#7958 (this issue here) only appeared in 0.22.1 (or maybe already in 0.22.0, not sure). Here the video does continue to play, but I cannot interact with the player. I cannot e.g. touch the screen to reveal the pause button. Or I cannot press the back button to tab out of the video. And if no video is playing, I cannot interact with any of the menus. They just act as if I hadn't touched them. The player resumes working once I close the split screen mode.

The only thing these two issues have in common is, that they both occur in split screen mode on my phone (Fairphone 4, Android 11).
